### PR TITLE
Remove system_platform_id from insights reports

### LINKED ISSUE #1993: Remove system_platform_id from insights report and logging
## Feature description:
Remove system_platform_id from insights report and logging.  Yupana now generates these IDs and the logging is confusing.

### PR DESCRIPTION
**Pull request by kholdaway**
Closes #1993

### CODE REVIEW

File: quipucords/api/insights_report/tests_insights_report.py [hunk 1]
```diff
@@ -56,7 +56,6 @@ def setUp(self):
                 'virt_num_guests': 1,
                 'virt_num_running_guests': 1,
                 'virt_what_type': 'vt',
-                'system_platform_id': self.sys_platform_id,
```
Comment (abaiken) [inline quipucords/api/insights_report/tests_insights_report.py]: looks like you could remove self.sys_platform_id

File: quipucords/api/reports/tests_reports.py [hunk 2]
```diff
@@ -165,6 +164,8 @@ def test_reports_gzip_renderer(self):
             file_contents = tar.extractfile(file).read().decode()
             if filenames[idx].endswith('csv'):
                 if 'details' in file_contents:
+                    print(repr(file_contents))
```
Comment (abaiken) [inline quipucords/api/reports/tests_reports.py]: remove debugging statements

File: quipucords/fingerprinter/task.py [hunk 3]
```diff
@@ -335,34 +334,26 @@ def _process_details_report(self, manager_interrupt, details_report):
...
-                    # If canonical facts, add it to the insights_hosts dict
-                    insights_id = fingerprint_dict.get(
-                        'system_platform_id')
                     if found_canonical_facts:
                         not_null_facts = {
                             'bios_uuid': 'bios_uuid',
                             'ip_addresses': 'ip_addresses',
                             'mac_addresses': 'mac_addresses',
-                            'insights_id': 'insights_client_id',
+                            'insights_client_id': 'insights_client_id',
```
Comment (abaiken) [inline quipucords/fingerprinter/task.py]: I think this should stay as `insights_id`

File: quipucords/fingerprinter/tests_fingerprint_task.py [hunk 4]
```diff
@@ -1143,7 +1143,7 @@ def test_process_details_report_success(self):
                     'dhcp181-3.gsslab.rdu2.redhat.com',
                 'fqdn': 'dhcp181-3.gsslab.rdu2.redhat.com',
                 'ip_addresses': ['1.2.3.4'],
-                'insights_id': '3f01b55457674041b75e41829bcee1dc',
+                'insights_client_id': '3f01b55457674041b75e41829bcee1dc',
```
Comment (abaiken) [inline quipucords/fingerprinter/tests_fingerprint_task.py]: Change this back

File: quipucords/api/reports/tests_reports.py (hunk 2 above)
Comment (kholdaway) [reply quipucords/api/reports/tests_reports.py]: Thanks!

File: quipucords/fingerprinter/task.py (hunk 3 above)
Comment (kholdaway) [reply quipucords/fingerprinter/task.py]: Yep

File: quipucords/fingerprinter/tests_fingerprint_task.py (hunk 4 above)
Comment (kholdaway) [reply quipucords/fingerprinter/tests_fingerprint_task.py]: okay 😨

**Pull request merged**